tests/pds/dataservice_test.go

```diff
@@ -98,6 +99,115 @@ var _ = Describe("{DeletePDSPods}", func() {
 	})
 })
 
+var _ = Describe("{UpdatePDSHelmVersion}", func() {
+	steplog := "Validate Data service pods post helm upgrade"
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("UpdatePDSHelmVersion", steplog, pdsLabels, 0)
+	})
+
+	It(steplog, func() {
+		var wlparams pdslib.WorkloadGenerationParams
+		var dataservices []PDSDataService
+
+		steplog = "Install older helm verison"
```

Comment:
No need of separate var for logging purposes.

Reply:
we need it, so that its used for logging purpose.